src/components/Label/Label.tsx

```diff
@@ -56,6 +56,7 @@ export interface LabelProps extends QAProps {
     size?: 'xs' | 's' | 'm';
     /** Browser title for Label */
     title?: string;
+    animation?: 'gradient';
```

Comment:
It should be `loading` prop like in Button

Reply:
was thinking about same property names with Skeleton, ok